pkg/kubelet/cm/container_manager.go

```diff
@@ -115,6 +115,9 @@ type ContainerManager interface {
 	// GetNodeAllocatableAbsolute returns the absolute value of Node Allocatable which is primarily useful for enforcement.
 	GetNodeAllocatableAbsolute() v1.ResourceList
 
+	// UnrepareResources unprepares pod resources
+	UnprepareResources(*v1.Pod) error
```

Comment:
I'm wondering if we should call this `PrepareDynamicResources()` and `UnprepareDynamicResources()` to make it clear its tied to the DRA logic.

Reply:
done in this PR: https://github.com/pohly/kubernetes/pull/24